packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -72,6 +72,10 @@ module.exports = function(proxy, allowedHost) {
     // WebpackDevServer is noisy by default so we emit custom message instead
     // by listening to the compiler events with `compiler.hooks[...].tap` calls above.
     quiet: true,
+    // WebpackDevServer has an option to write every build to disk. This can be
+    // useful for certain use cases such as developing browser extensions where one
+    // would need to otherwise do full builds.
+    writeToDisk: false || process.env.WRITE_TO_DISK === 'true',
```

Comment:
We need to update it to comply with this https://github.com/facebook/create-react-app/pull/6144#issuecomment-452686869.

Reply:
I'm resolving this based on https://github.com/facebook/create-react-app/pull/6144#issuecomment-452725723